deploy/azureDeployFunction.js

```diff
@@ -18,6 +18,7 @@ class AzureDeployFunction {
 
     this.hooks = {
       'deploy:function:deploy': () => BbPromise.bind(this)
+        .then(this.provider.initialise(this.serverless,this.options))
```

Comment:
Is calling initialization before every command the only way to make sure serverless framework variables are properly initialized?